src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -115,7 +115,8 @@ public async Task<IActionResult> Index(ContentOptions options, PagerParameters p
                 new SelectListItem(S["Delete"], nameof(ContentsBulkAction.Remove)),
             ];
 
-            return View(model);
+            // Both the Templates and Admin Templates use the same view (Index.cshtml) but with a different route.
```

Comment:
@microposmp 

Can you please update the comment to use this instead?

// Both the 'Template' and 'Admin' actions render the 'Index' template.
// To ensure we render the same view, we explicitly specify the name of the view that should be rendered.